tutorials/diffusion_calculation.ipynb

```diff
@@ -0,0 +1,293 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Display the central slice of each map\n",
+    "fig, ax = plt.subplots(2, 2, figsize=(8, 6))\n",
+    "\n",
+    "# Display a central slice of the ADC map\n",
+    "im = ax[0, 0].imshow(np.rot90(adc_mapper.adc[:, :, adc_mapper.shape[2]//2]), cmap='inferno', clim=(0.001, 0.003))\n",
+    "cb = fig.colorbar(im, ax=ax[0, 0])\n",
+    "cb.set_label('ADC ($mm^2/s$)')\n",
+    "ax[0, 0].axis('off')\n",
+    "\n",
+    "# Display a central slice of the MD map\n",
+    "im = ax[0, 1].imshow(np.rot90(dti_mapper.md[:, :, dti_mapper.shape[2]//2]), cmap='inferno', clim=(0.001, 0.003))\n",
+    "cb = fig.colorbar(im, ax=ax[0, 1])\n",
+    "cb.set_label('MD ($mm^2/s$)')\n",
+    "ax[0, 1].axis('off')\n",
+    "\n",
+    "# Display a central slice of the FA map\n",
+    "im = ax[1, 0].imshow(np.rot90(dti_mapper.fa[:, :, dti_mapper.shape[2]//2]), cmap='viridis', clim=(0.1, 0.8))\n",
+    "cb = fig.colorbar(im, ax=ax[1, 0])\n",
+    "cb.set_label('FA')\n",
+    "ax[1, 0].axis('off')\n",
+    "\n",
+    "# Display a central slice of the color FA map\n",
+    "im = ax[1, 1].imshow(np.rot90(dti_mapper.color_fa[:, :, dti_mapper.shape[2]//2, :]))\n",
```

Comment:
Is there a colorbar for color_fa? You described it correctly in diffusion.py, but I guess it doesn't apply here because it involves the 3 colours channels (R, G and B)?

I guess experts will understand this? I'm no expert in diffusion at all, but seeing this specific plot it reminds me of those very "cool" 3D brain DTI colour maps. So I guess they all represent the same?

Reply:
Yeah, you don't get colour bars on those types of plots. The closest people do is the example below. I made this one in Photoshop and it was enough of a faff there, adding it to a matplotlib subplot would be a major pain. Most people that deal with diffusion data know what the colours just represent direction.
<img src="https://user-images.githubusercontent.com/23356429/117315150-52e8cc00-ae7f-11eb-81e8-1a27a9a71fbc.png" alt="DTI with rose" width="300"/>

Reply:
Really appreciate spending your time in Photoshop to create this picture :) Thanks for the clarification!